CHANGELOG.md

```diff
@@ -5,6 +5,11 @@ The format is based on [Keep a Changelog](http://keepachangelog.com/).
 
 ## [Unreleased]
 
+## [2.0] - 2018-09-21
```

Comment:
Pls put your changes to the `Unreleased` section. I gonna update this during the release process...

Reply:
it is fixed.